engine/src/main.cpp

```diff
@@ -24,9 +32,20 @@ int main()
     FUNCT_ERROR("functError");
     FUNCT_WARNING("functWarning");
 
-    GPE_ASSERT(false, "GPE_assertInfo");
+    GPE_ASSERT(true, "GPE_assertInfo");
+}
+
+int main()
+{
+    // Log::setSetting(ESetting::ALWAYS_PRINT_LOG_FILE, true);
+
+    WindowGLFW win(WindowCreateArg{"GP engine", 600, 900});
+    RendererGLFW_GL46 ren (win);
```

Comment:
I'm being picky: there is an extra space here
```suggestion
    RendererGLFW_GL46 ren(win);
```